### DISCUSSION TITLE
Better highlighting the active tab (dark mode)?

### DISCUSSION THREAD

**Discussion by jnachtigall** [piroor/treestyletab#3356] [Q&A]
I've always had problem finding the active tabs. Is there a ways to better highlight the active tab? For instance by giving more contrast to the back/foreground or by having a lighter background color for it?

Here's an example screenshot. I really have problem finding/seeing my active tab:
<img src="https://github.com/piroor/treestyletab/assets/3307404/bc84d916-083c-4472-92b0-ae815b30d2dd" width="25%">

**Comment by irvinm** [comment]
How about something like this ... change to your taste.

```
tab-item.active tab-item-substance { 
   background: orange;
} 
```

![image](https://github.com/piroor/treestyletab/assets/979729/dd8dc307-0566-40e2-adbd-9d3c28dade1f)
